packages/siopv2-oid4vp-op-auth/package.json

```diff
@@ -17,22 +17,31 @@
     "@sphereon/did-auth-siop": "0.6.4",
     "@sphereon/pex": "^3.3.3",
     "@sphereon/pex-models": "2.2.4",
+    "@sphereon/ssi-sdk-ext.did-resolver-jwk": "^0.21.0",
```

Comment:
Why does it depend on did-jwk resolution all of a sudden. That should be in dev deps

Reply:
Ping

Reply:
```
import { KeyUse } from '@sphereon/ssi-sdk-ext.did-resolver-jwk'
```
It was already moved to devDeps, it's just for the enum